src/plugins/kibana/public/settings/sections/indices/add_data_steps/pattern_review_step.html

```diff
@@ -1,7 +1,28 @@
-<h2>Pattern review step</h2>
+<div class="wizard-step-title">
+  <h3>Review the index pattern</h3>
+  Here we'll define how and where to store your parsed events. We've made some intellient guesses for you, but most
+  fields can be changed if we got it wrong!
+</div>
 
-<div>
-  Docs: {{sampleDocs}}
+<div class="pattern-review">
+  <input ng-model="reviewStep.indexPattern.id" class="pattern-input"/>
```

Comment:
Added a label, and disabled the save button if the pattern is empty. Spencer brought up the idea of using Angular's form validation as well, when reviewing the wizard shell PR. I think it's a good idea but I'm going to tackle it in another pull so that I can convert the entire wizard into a form, with each step being a sub-form, which will be a part of a larger effort to make the wizard itself more reusable.